docs/developer-guide/install.md

```diff
@@ -0,0 +1,74 @@
+---
+order: 2
+description: Steps to install Starport on your local computer.
+---
+
+# Install Starport
+
+You can run Starport in a web-based Gitpod IDE or you can install Starport on your local computer.
+
+## Prerequisite
+
+Starport is written in the Go programming language. To use Starport on a local system:
+
+- Install [Go](https://golang.org/doc/install) (**version 1.16** or higher)
+- Ensure the Go environment variables are [set properly](https://golang.org/doc/gopath_code#GOPATH) on your system
+
+**Tip:** If you get the message `appd: command not found` then your Go environment variables might not be set correctly.
+
+## Upgrading Your Starport Installation
```

Comment:
IMO, the upgrading and removing should be it's own topic?

Reply:
it looks good to me in the installation guide for now :)